TheAlienThatEatsTheCarrot/GameEngine/Components/GameComponents/MovableComponent.swift

```diff
@@ -10,11 +10,11 @@ import Foundation
 class MovableComponent: Component {
     var entity: Entity
     var direction: Direction
-    var distance: CGFloat
+    var velocity: CGFloat
```

Comment:
would it be better to have it as CGVector for the velocity? since it may move in both x and y direction?

Reply:
Right, I missed that part. Thanks for noticing!